.github/workflows/lint.yml

```diff
@@ -8,10 +8,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up Python 3.10
+      - name: Set up Python 3.9
```

Comment:
Why downgrade this version?

Reply:
it is an upgrade. I think we will deprecate python 3.8 soon

Reply:
OK, I was mistaken because looking at this commit alone, it seems like 3.10 -> 3.9, but in reality, it's from 3.8 -> 3.9.